docs/src/configuration.md

```diff
@@ -170,3 +170,22 @@ WIP
 ]
 ```
 ````
+
+## `.gitignore`
+
+### Retaining files when step is reset
+
+Reseting step will remove all untracked files from the project working directory (`<PROJECT_DASHED_NAME>`). `.gitignore` files can be used to prevent git from removing certain files when step is reset. Normal gitignore patterns applies.
```

Comment:
````suggestion
```admonish warning
Resetting a step removes all untracked files from the project directory. To prevent this for specific files, add them to the `.gitignore` file in the root; `.gitignore` files in the project directory are also removed upon reset.
```
````

Reply:
> `.gitignore` files in the project directory are also removed upon reset.

Uhm, if `.gitignore` in the project directory is committed, then it's not going to 
 be removed on the reset.

Reply:
Right, provided they are a part of the boilerplate - not added after the fact.

I have updates the suggestion